mod/wall_upload.php

```diff
@@ -119,6 +119,7 @@ function wall_upload_post(&$a, $desktopmode = true) {
 	$maximagesize = get_config('system','maximagesize');
 
 	if(($maximagesize) && ($filesize > $maximagesize)) {
+		logger("Image exceeds size limit of ".$maximagesize);
 		echo  sprintf( t('Image exceeds size limit of %s'), formatBytes($maximagesize)) . EOL;
```

Comment:
The function is used via frontend and via backend (with the api). So the logger is a good idea. This line is the rest of some test lines I created, since I yesterday had several problems, uploading pictures. It is unimportant for the OStatus communication in this pull request. So I could remove the changed line if you want. Then we can work on it at a later time.